prow/config/config.yaml

```diff
@@ -2295,6 +2390,7 @@ tide:
         - release-7.1 # LTS: avoid prow's "Merging to branch *** is forbidden" description issue.
         - release-7.2 # DMR: avoid prow's "Merging to branch *** is forbidden" description issue.
         - release-7.3 # DMR: avoid prow's "Merging to branch *** is forbidden" description issue.
```

Comment:
We can delete the these released DMR branches.

Reply:
We just need to keep the latest DMR branches

Reply:
Will create a separate PR to handle the deletion of DMR-related content.